nightly/Dockerfile

```diff
@@ -6,9 +6,9 @@ ENV KRE_USER_HOME /opt/kre
 RUN apt-get -qq update && apt-get -qqy install unzip
 
 ONBUILD RUN curl -sSL https://raw.githubusercontent.com/aspnet/Home/dev/kvminstall.sh | sh
-ONBUILD RUN bash -c "source $KRE_USER_HOME/kvm/kvm.sh \
+ONBUILD RUN bash -c "KVM_USER_HOME=$KRE_USER_HOME && source $KRE_USER_HOME/kvm/kvm.sh \
 	&& kvm install latest -a default \
-	&& kvm alias default | xargs -i ln -s $KRE_USER_HOME/packages/{} $KRE_USER_HOME/packages/default"
+	&& kvm alias default | xargs -i ln -s $KRE_USER_HOME/runtimes/{} $KRE_USER_HOME/runtimes/default"
```

Comment:
is the packages→runtimes change something comes with newer versions of kvm? I haven't really looked into these.